objects/rct2tt/ride/rct2.tt.cyclopsx.json

```diff
@@ -39,7 +39,7 @@
     "images": ["$RCT2:OBJDATA/CYCLOPSX.DAT[0..66]"],
     "strings": {
         "name": {
-            "en-GB": "Cyclops Ride",
+            "en-GB": "Cyclops Cars",
```

Comment:
These rides are listed as independent rides in the New Ride window, hence the `Cyclops Ride` name. Not sure if it should be changed.